src/sensor_ads1220.c

```diff
@@ -80,6 +82,7 @@ add_sample(struct ads1220_adc *ads1220, uint8_t oid, uint_fast32_t counts)
 void
 ads1220_read_adc(struct ads1220_adc *ads1220, uint8_t oid)
 {
+    uint32_t start = timer_read_time();
```

Comment:
Not a blocker, but for what it is worth, `timer_read_time()` is expensive on some micro-controllers.  Might be preferable to have `load_cell_probe_report_sample()` call `timer_read_time()` itself after it confirms homing is active.  (Admittedly, that may not be the most accurate timestamp, but no timestamp is particularly accurate - which is why we ultimately want tap analysis.)

Reply:
Fair, now that we have TapAnalysis this time is not really used. Removed.